addons/mail/static/tests/discuss_app/jump_to_present.test.js

```diff
@@ -260,3 +260,25 @@ test("show jump to present banner after scrolling up 10 messages", async () => {
     await scroll(".o-mail-Thread", queryFirst(".o-mail-Thread").scrollTop - 5 * messageHeight);
     await contains("[title='Jump to Present']");
 });
+
+test("focus composer after jump to present", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "General" });
+    pyEnv["mail.message"].create(
+        [...Array(40).keys()].map((i) => ({
+            body: `<p>Non Empty Message ${i}</p>`,
+            message_type: "comment",
+            model: "discuss.channel",
+            res_id: channelId,
+        }))
+    );
+    await start();
+    await openDiscuss(channelId);
+    await contains(".o-mail-Message", { count: 30 });
+    await scroll(".o-mail-Thread", 0);
+    await contains(".o-mail-Composer.o-focused");
```

Comment:
`await scroll(".o-mail-Thread", 0);` looks useless: the conversation is already at the top, since self member has no `new_message_separator` defined and thus it implicitly means all messages are considered new.

In this case, the "auto-scroll" is above the 1st unread message, which is already the top of conversation.